Linemux does not compile anymore -- notify has been updated and broke it.

```
   Compiling linemux v0.2.3
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> /home/sjm/.cargo/registry/src/github.com-1ecc6299db9ec823/linemux-0.2.3/src/events.rs:62:49
    |
62  |           let inner: notify::RecommendedWatcher = notify::RecommendedWatcher::new(move |res| {
    |  _________________________________________________^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^-
63  | |             // The only way `send` can fail is if the receiver is dropped,
64  | |             // and `MuxedEvents` controls both. `unwrap` is not used,
65  | |             // however, since `Drop` idiosyncrasies could otherwise result
66  | |             // in a panic.
67  | |             let _ = tx.send(res);
68  | |         })
    | |__________- an argument of type `Config` is missing
    |
note: associated function defined here
   --> /home/sjm/.cargo/registry/src/github.com-1ecc6299db9ec823/notify-5.0.0-pre.16/src/lib.rs:294:8
    |
294 |     fn new<F: EventHandler>(event_handler: F, config: config::Config) -> Result<Self>
    |        ^^^
help: provide the argument
    |
62  ~         let inner: notify::RecommendedWatcher = notify::RecommendedWatcher::new(move |res| {
63  +             // The only way `send` can fail is if the receiver is dropped,
64  +             // and `MuxedEvents` controls both. `unwrap` is not used,
65  +             // however, since `Drop` idiosyncrasies could otherwise result
66  +             // in a panic.
67  +             let _ = tx.send(res);
68  +         }, /* Config */)
    |

For more information about this error, try `rustc --explain E0061`.
error: could not compile `linemux` due to previous error
```

Well, as `linemux` commit from 2022-06-05 says "update notify to 5.0.0-pre.14" I suspect that now `notify` has made a breaking change with their pre5-version.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org).
 
 ### Changed
 - Muxed events: include paths in remove events
-- Update `notify` to `5.0.0-pre.14` to allow `kqueue` support
+- Update `notify` to `5.0.0-pre.16` to allow `kqueue` support
 - Switch OSX notification backend from `fsevents` to `kqueue`
 - Store `Watcher` as a trait object to allow runtime variance of the
   notification backend.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,7 +21,7 @@ tokio = ["tokio_"]
 
 [dependencies]
 futures-util = { version = "0.3", default-features = false, features = ["std"] }
-notify = { version = "5.0.0-pre.14", default-features = false, features = ["macos_kqueue"] }
+notify = { version = "=5.0.0-pre.16", default-features = false, features = ["macos_kqueue", "crossbeam-channel"] }
 pin-project-lite = "0.2"
 tokio_ = { package = "tokio", version = "1", features = ["fs", "io-util", "sync", "time"], optional = true }
 

--- a/src/events.rs
+++ b/src/events.rs
@@ -60,13 +60,16 @@ impl MuxedEvents {
     /// Constructs a new `MuxedEvents` instance.
     pub fn new() -> io::Result<Self> {
         let (tx, rx) = mpsc::unbounded_channel();
-        let inner: notify::RecommendedWatcher = notify::RecommendedWatcher::new(move |res| {
-            // The only way `send` can fail is if the receiver is dropped,
-            // and `MuxedEvents` controls both. `unwrap` is not used,
-            // however, since `Drop` idiosyncrasies could otherwise result
-            // in a panic.
-            let _ = tx.send(res);
-        })
+        let inner: notify::RecommendedWatcher = notify::RecommendedWatcher::new(
+            move |res| {
+                // The only way `send` can fail is if the receiver is dropped,
+                // and `MuxedEvents` controls both. `unwrap` is not used,
+                // however, since `Drop` idiosyncrasies could otherwise result
+                // in a panic.
+                let _ = tx.send(res);
+            },
+            notify::Config::default(),
+        )
         .map_err(notify_to_io_error)?;
 
         Ok(MuxedEvents {

